instruqt-tracks/consul-life-of-a-developer/track_scripts/setup-workstation

```diff
@@ -241,4 +241,10 @@ if [ "$leader" != "200" ]; then
   exit 1
 fi
 
+#creating kubectl namespaces to default to "default"
+kubectl config use-context k8s1
+kubectl config set-context --current --namespace=default 
+kubectl config use-context k8s2
+kubectl config set-context --current --namespace=default 
```

Comment:
Rather than edit multiple kubeconfig files directly this way, it may be simpler (and more portable to other tracks) to unset the KUBERNETES_* environment variables that are causing the namespace info in the in-cluster kubeconfig to be used:

```
for kubevar in $(env | grep KUBE | cut -d= -f1); do unset $kubevar; done
```

You can add that to the workstation track script and it should take care of this issue at the source.

Reply:
Thank you @omkensey  - i will test this out and open a new PR.